### PR TITLE
Closes #9

### LINKED ISSUE #9: Add CLI args to select tone row or column
Currently, the `twelve-tone` CLI generates a random melody and takes the top row as it's tone row.
https://github.com/accraze/python-twelve-tone/blob/master/src/twelve_tone/cli.py#L26

There should be some CLI flags that allow you to specify which row or column you wish to use for the generated melody. This should be pretty easy, just update the main to be something like:

```
@click.command()
@click.option('--row', '-r',  default=0, help='Which row should we use?')
@click.option('--col', '-c',  default=None, help='Which column should we use?')
def main(row,column):
    c = Composer()
    c.compose()
    click.echo(c.get_melody(row=row, column=column))
```

### PR DESCRIPTION
**Pull request by rhythmize**
Added cli arguments to click to specify row and column number for selecting row or column tone

### CODE REVIEW

File: src/twelve_tone/cli.py [hunk 1]
```diff
@@ -20,7 +20,18 @@
...
 @click.command()
-def main():
+@click.option('--row', '-r', default=0, help='Row to use as row tone')
+@click.option('--column', '-c', default=0, help='Column to use as column tone')
+def main(row, column):
     c = Composer()
     c.compose()
-    click.echo(c.get_melody())
+    if row < 0 or column < 0:
+      click.echo("Invalid row or column arguments.")
```
Comment (accraze) [inline src/twelve_tone/cli.py]: Can you indent to 4 spaces here instead of two?

File: src/twelve_tone/cli.py [hunk 2]
```diff
@@ -20,7 +20,18 @@
...
+      exit(1)
+    elif row >= c.matrix.shape[0]:
+      click.echo("Row number exceeds melody row count.")
```
Comment (accraze) [inline src/twelve_tone/cli.py]: Indent to 4 spaces here instead of two please

File: src/twelve_tone/cli.py [hunk 3]
```diff
@@ -20,7 +20,18 @@
...
+      exit(1)
+    elif column >= c.matrix.shape[1]:
+      click.echo("Column number exceeds melody column count.")
```
Comment (accraze) [inline src/twelve_tone/cli.py]: Indent to 4 spaces here instead of two please

**Review by accraze — COMMENTED** [review]
Hey @code-player, thanks for the PR! Looks good, minus a couple of indent issues. Can you please fix and I will merge it.

Here is the error in the build CI: 
https://travis-ci.org/accraze/python-twelve-tone/jobs/436017508#L532

**Comment by rhythmize** [comment]
done.

**Review by accraze — APPROVED** [review]
Awesome thanks for the PR @code-player! Happy Hacktoberfest 🎃 🖥

**Pull request merged**